sponsorships/invoice.md

```diff
@@ -17,21 +17,22 @@ Tier X Conference Sponsorship - 2019
 
 ## Notes
 
-PAYMENT OPTIONS
-1. By mail with cheque
-2. Through your financial institution with EFT or wire, for wire forms from our financial insitution see: https://share.mayfirst.org/s/9nNnFPia89GHcod
-
-BENEFICIARY BANK (SWIFT FIELD 57)
-Alterna Savings and Credit Union Limited
-319 McRae Avenue, 1st floor
-Ottawa, ON, Canada  K1Z 0B9
-Transit No.: 00646-842
-
-BENEFICIARY CUSTOMER (SWIFT FIELD 59)
-Account No.: [OUR_NETWORKS_ACCOUNT_NO]
-Member Name: Our Networks
-Member Mailing Address: [OUR_NETWORKS_CONTACT_ADDRESS]
-
-MAILING ADDRESS
-c/o [OUR_NETWORKS_CONTACT_NAME]
-[OUR_NETWORKS_CONTACT_ADDRESS]
+PAYMENT OPTIONS  
+1. By mail with cheque payable to Our Networks  
+2. Through your financial institution with EFT or wire (see attached form)  
+
+FINANCIAL INSTITUTION  
+Alterna Savings and Credit Union Limited  
+319 McRae Avenue, 1st floor  
```

Comment:
We used to put the `800 Bay Street` address

Reply:
yes, I realized that we should use the main address for all of them -- verified this Ottawa address comes up in another Canadian bank transfer screen (ATB)